src/components/SidebarGroup/SidebarGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import SidebarGroupHeader from './SidebarGroupHeader/SidebarGroupHeader'
+import SidebarGroupComponent from './SidebarGroupComponent/SidebarGroupComponent'
+import { Header } from '../../stories/Header'
+
+export interface SidebarGroupProps {
```

Comment:
To make this more flexible and intuitive, it should be able to take a variable number items. It should also be able to handle clicks with functions that are passed in. The interface might look more like: 
```ts
enum ItemStatus {
  Active,
  Inactive,
  Completed
}

type SidebarGroupItemProps {
  status: ItemStatus,
  value: string,
  action: () => void
}

type SidebarGroupProps{
  header: string,
  items: SidebarGroupItemProps[]
} 
```

From here, you can enumerate the `props.items` in JSX and create a component for each entry.